core/src/main/java/nl/nn/adapterframework/webcontrol/api/TransactionalStorage.java

```diff
@@ -140,6 +140,7 @@ private StorageItemDTO getMessageMetadata(IMessageBrowser<?> storage, String mes
 
 	public static class StorageItemDTO {
 		private @Getter String id; //MessageId
+		private @Getter String messageId; // just to display truncated data
```

Comment:
Could you use descriptive names here, like 'id' and 'displayedId'

Reply:
Id and messageid is the same?!